sdk/core/azure-core/tests/test_user_agent_policy.py

```diff
@@ -0,0 +1,25 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Tests for the user agent policy."""
+from azure.core.pipeline.policies import UserAgentPolicy
+from azure.core.pipeline.transport import HttpRequest
+from azure.core.pipeline import PipelineRequest, PipelineContext
+
+def test_user_agent_policy():
```

Comment:
There is a test_user_agent in test_universal_pipeline please move it here and make the tests consistent